packages/nimble-components/src/table/tests/table-column-sizing.spec.ts

```diff
@@ -321,3 +347,487 @@ describe('Table Column Sizing', () => {
         }
     });
 });
+
+describe('Table Interactive Column Sizing', () => {
+    let element: Table<SimpleTableRecord>;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+    let pageObject: TablePageObject<SimpleTableRecord>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setupInteractiveTests());
+        pageObject = new TablePageObject<SimpleTableRecord>(element);
+        await connect();
+        await element.setData(simpleTableData);
+        await waitForUpdatesAsync();
+        await pageObject.sizeTableToGivenRowWidth(400, element);
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    describe('No hidden columns ', () => {
+        const columnSizeTests = [
```

Comment:
Is there a test that covers moving the mouse farther to the left than can columns can shrink, then moving the mouse to the right a little bit and ensuring column widths don't change (i.e. that the divider gets re-synced to the cursor's position)? I didn't see one, but it's quite possible I missed it. If there isn't one, could you add one?

Reply:
Added test.